src/shared/actions/archives.js

```diff
@@ -49,7 +49,7 @@ export const unlockArchive = payload => dispatch => {
     password => unlockArchiveInArchiveManager(payload, password)
   ).then(
     archiveId => dispatch(loadArchive(archiveId))
-  );
+  ).catch(() => {});
```

Comment:
Should this dispatch an error message?

Reply:
No, this catch is only for the password dialog "cancel" event.

Reply:
What if this fails? `unlockArchiveInArchiveManager(payload, password)`

Reply:
That is handled by the dialog (it shows the error message). The dialog is a but weird